src/NServiceBus.NHibernate.TransactionalSession/NHibernateTransactionalSessionOptionsCustomization.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace NServiceBus.TransactionalSession;
+
+class NHibernateTransactionalSessionOptionsCustomization : IOpenSessionOptionsCustomization
+{
+    readonly string endpointName;
+
+    public NHibernateTransactionalSessionOptionsCustomization(string endpointName) => this.endpointName = endpointName;
```

Comment:
Technically you could also inject readonly settings and then from there get the endpoint name. Both approaches are valid. Just wanted to mention it

Reply:
I wanted to make it explicit what is exactly (in terms of settings) that we need in the extension.